javascript/ql/lib/semmle/javascript/security/dataflow/UnsafeShellCommandConstructionQuery.qll

```diff
@@ -24,7 +24,9 @@ class Configuration extends TaintTracking::Configuration {
 
   override predicate isSanitizerGuard(TaintTracking::SanitizerGuardNode guard) {
     guard instanceof PathExistsSanitizerGuard or
-    guard instanceof TaintTracking::AdHocWhitelistCheckSanitizer
+    guard instanceof TaintTracking::AdHocWhitelistCheckSanitizer or
+    guard instanceof NumberGuard or
+    guard instanceof TypeOfSanitizer
```

Comment:
Is this `TypeOfSanitizer` supposed to be a part of this PR?

Reply:
Yes, I noticed that we defined the guard in the Customizations file, but we never included it in the `isSanitizerGuard` predicate.   
I did a similar thing in `UnvalidatedDynamicMethodCallQuery`.